VetClinic.BLL.Tests/AutoMoqDataAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoFixture;
+using AutoFixture.AutoMoq;
+using AutoFixture.Xunit2;
+using System.Linq;
+
+namespace VetClinic.BLL.Tests
+{
+    class AutoMoqDataAttribute : AutoDataAttribute
+    {
+        public AutoMoqDataAttribute() : base(() => {
+            var fixture = new Fixture().Customize(new AutoMoqCustomization());
+
+            fixture.Behaviors.OfType<ThrowingRecursionBehavior>().ToList().ForEach(b => fixture.Behaviors.Remove(b));
+            fixture.Behaviors.Add(new OmitOnRecursionBehavior());
+
+            return fixture;
+        })
+        {
+        
```

Comment:
Why do you need the current block?

Reply:
It's just an empty costructor, all the magic happens in the base(...). Or do you mean that i should return it inline?